### ISSUE TITLE
Disable preview for tab completion

### ISSUE THREAD

**Issue by daephx** [kelleyma49/PSFzf#264]
With my setup the preview window for tab completion is too vertically narrow to be of any real use and I am not a fan of it being enabled by default.

Looking around the docs and code It's not clear to me if this is even possible with the current implementation.
If this is in-fact possible, I would appreciate some pointers. Otherwise I think this is a issue.

**Comment by YangHanlin** [comment]
Same issue here. The preview window causes some lag on my machine, but I can't find any option to disable it.

**Comment by marco9446** [comment]
You can achieve that by setting an option for fzf itself.
In my powershell profile i have 
```shell
[Environment]::SetEnvironmentVariable("FZF_DEFAULT_OPTS", "--preview-window=hidden")
```

**Comment by daephx** [comment]
That works if you just want to remove it completely and your config is fairly simple.
But it's not a fix in my option. I don't experience much lag from its presence. But the fact it is positioned at the bottom or on by default at all is not to my liking.

**Comment by LucasOe** [comment]
I'm using this workaround:
```ps1
# Overwrites the preview-window options using `bind='start:'`
$env:_PSFZF_FZF_DEFAULT_OPTS = "`
    $env:FZF_DEFAULT_OPTS `
    --bind='start:change-preview-window(hidden)+unbind(Ctrl-/)'"
```

It's a bit dirty, but this way I can keep the preview window for `FZF_DEFAULT_OPTS`, but disable it for `_PSFZF_FZF_DEFAULT_OPTS`.

**Comment by L1ttleFlyyy** [comment]
First of all I wanna thank you for bringing this useful script to powershell! I am not here to complain, just to give some feedback, with vivid examples.

For me I do wanna use the preview but the default behavior is not usable.
Also I couldn't seem to be able to tweak the preview settings via FZF_DEFAULT_OPTS.
- If I don't set anything in `FZF_DEFAULT_OPTS`, and give `Psfzf` the full screen, the preview barely shows 3 lines of content:
<img src="https://github.com/user-attachments/assets/9e4f381e-b6cc-4ffa-96db-cb94a4b397dd" width="60%">

- If I set to `$Env:FZF_DEFAULT_OPTS="--height=70%`, then it is basically useless just wasting display space
<img src="https://github.com/user-attachments/assets/46fac452-f914-4370-9092-96ba8e4603d1" width="60%">

Even if I toggle to vertical preview with `ctrl-/`, it is still barely shows the filename because of the Get-ChildItem formatting:
<img src="https://github.com/user-attachments/assets/2e5ec6dc-63bc-4266-a81c-4779ce6e9b46" width="60%">

The `--preview-windows` setting doesn't seem to be working. `$Env:FZF_DEFAULT_OPTS="--height=70% --preview-window=right:60%`. Seems like `Psfzf` is overriding it internally.

direct calling fzf with `preview-window=right:60%`
<img src="https://github.com/user-attachments/assets/68052daa-c7f2-4512-a53d-e291af4c0d91" width="60%">

`Psfzf` with `preview-window=right:60%`:
<img src="https://github.com/user-attachments/assets/627a68fc-fde7-4081-9418-b348e0a85dc8" width="60%">

So basically we need 2 improvements here:
- the default `Psfzf` preview setting needs to be tweaked for Get-ChildItem, the default format is too sparse for preview window
- exposing a user config to disable/tweak `Psfzf`'s preview window

**Comment by sebastianstudniczek** [comment]
> I'm using this workaround:
> 
> # Overwrites the preview-window options using `bind='start:'`
> $env:_PSFZF_FZF_DEFAULT_OPTS = "$env:FZF_DEFAULT_OPTS --bind='start:change-preview-window(hidden)+unbind(Ctrl-/)'"
> It's a bit dirty, but this way I can keep the preview window for `FZF_DEFAULT_OPTS`, but disable it for `_PSFZF_FZF_DEFAULT_OPTS`.

It possible to still have binded `ctrl-/` to enable preview window?

**Comment by kelleyma49** [comment]
I basically disabled this on startup. You can enable preview via the keyboard shortcut <kbd>CTRL+/</kbd>. 

Please try this out in this version: https://www.powershellgallery.com/packages/PSFzf/2.7.1-alpha

**Comment by mrbeardad** [comment]
> I basically disabled this on startup. You can enable preview via the keyboard shortcut CTRL+/.
> 
> Please try this out in this version: https://www.powershellgallery.com/packages/PSFzf/2.7.1-alpha

@kelleyma49 but how to enable it by default? I don't want to press CTRL+/

**Comment by mrbeardad** [comment]
> [@mrbeardad](https://github.com/mrbeardad) , I've added the ability in this PR:
> 
> [#361](https://github.com/kelleyma49/PSFzf/pull/361)
> 
> Try it in this prerelease:
> 
> https://www.powershellgallery.com/packages/PSFzf/2.7.6-alpha

@kelleyma49 Yes, it works well, thanks!
